### DISCUSSION TITLE
STEAM LINK

### DISCUSSION THREAD

**Discussion by cjjtz** [elrumo/macOS_Big_Sur_icons_replacements#1122] [Icon request]
Want a new icon called "steamlink"
<img width="148" alt="截屏2021-03-21 11 51 32" src="https://user-images.githubusercontent.com/74545743/111893196-eb480000-8a3b-11eb-9906-bf78e67c2540.png">

**Comment by cake03** [comment]
### Already exists

   Official icon:
> ![](https://is2-ssl.mzstatic.com/image/thumb/Purple115/v4/2e/8c/e5/2e8ce586-686a-7d2e-8bd9-dbcac89b7657/icon.png/460x0w.webp)

**Reply by cake03** [reply]
You may need to update the app for the change to reflect
